package.json

```diff
@@ -8,18 +8,18 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/anmolbabu/manager-api.git"
+    "url": "git+https://github.com/fabric8-hdd/fabric8-hypothesis-manager-api.git"
   },
   "keywords": [],
   "author": "Anmol Babu <anmolbudugutta@gmail.com>",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/anmolbabu/manager-api/issues"
+    "url": "https://github.com/fabric8-hdd/fabric8-hypothesis-manager-api/issues"
   },
-  "homepage": "https://github.com/anmolbabu/manager-api#readme",
+  "homepage": "https://github.com/fabric8-hdd/fabric8-hypothesis-manager-api#readme",
   "dependencies": {
     "validator": "^9.2.0",
-    "hypothesis-core": "github:fabric8-hdd/hypothesis-core"
+    "hypothesis-core": "github:fabric8-hdd/fabric8-hypothesis-core"
```

Comment:
Hey @anmolbabu , is the npm install working properly for you here?

Try npm install && ls node_modules/ | grep fabric8

and see if there's any module installed.

Doesn't work in my case. So I've changed to "fabric8-hypothesis-core": "github:fabric8-hdd/fabric8-hypothesis-core"

Reply:
well "github:fabric8-hdd/fabric8-hypothesis-core" didn't work for me.

So I had to manually do npm install fabric8-hdd/fabric8-hypothesis-core which created this line for me and thereafter deleting node_modules and package-lock.json and doing npm install actually pulled in the dependency...

I can give it another try since you are saying this doesn't work for you..